src/Security/AccessResolver/AbstractAccessResolver.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace WhiteDigital\EntityResourceMapper\Security\AccessResolver;
+
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\QueryBuilder;
+use InvalidArgumentException;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+use WhiteDigital\EntityResourceMapper\Entity\BaseEntity;
+use WhiteDigital\EntityResourceMapper\Security\Attribute\AccessResolverConfiguration;
+use WhiteDigital\EntityResourceMapper\Security\Interface\AccessResolverInterface;
+
+abstract class AbstractAccessResolver implements AccessResolverInterface
```

Comment:
Šis īsti neatbilst patiesībai. AccessResolver var būt pilnīgi jebkas. Tikpat labi AccessResolver var pārbaudīt, ja ir pāra datums, tad neatļaut...
```suggestion
abstract class AbstractOwnerPropertyBasedAccessResolver implements AccessResolverInterface
```

Reply:
nu tad tas varētu implementēt AccessResolverInterface nevis extendot šo. jo nu gan tas owner, gan role, gan tas, ko es zero izmantoju, lieliski atbilst šai abstrakcijai. nosaukt viņu citādāk varbūt?